src/SwedbankPay/Core/Library/Methods/Checkout.php

```diff
@@ -10,6 +10,48 @@
 
 trait Checkout
 {
+	/**
+	 * Check Checkout API Credentials.
+	 *
+	 * @return bool Returns true if ok
+	 * @throws Exception
+	 */
+	public function checkCheckoutApiCredentials()
+	{
+		/**
+		 * Perform a POST request to create a payment or paymentorder with invalid data that is known to fail with
+		 * 400 Bad Request.
+		 * This is to ensure that no payments or payment orders are actually created if the request succeeds.
+		 * If the request fails with 401, something is wrong with the credentials.
+		 * If the request fails with 403 something is wrong with the contract.
+		 * If the request fails with 400 Bad Request, credentials and contracts should be OK.
+		 */
+
+		$params = [
+			'paymentorder' => [
+				'operation' => 'Test',
+				'payeeInfo' => [
+					'payeeId' => $this->getConfiguration()->getPayeeId(),
+					'payeeName' => $this->getConfiguration()->getPayeeName(),
+				]
+			]
+		];
+
+		try {
+			$this->request('POST', '/psp/paymentorders', $params);
+		} catch (Exception $e) {
+			if (400 === $e->getCode()) {
```

Comment:
Please have a look inside `problems` to see that the `operation` was the field with error and not something else. If the status code is `400` and `operation` can't be found in the `problems` array, throw an exception with content from the `application/problem+json` response. I think the SDK should have a utility method to convert an `application/problem+json` response to an exception.

Reply:
@asbjornu It returns empty

Reply:
@asbjornu It doesn't matter because we just need to check response code